### PR TITLE
[TASK] Improve code quality by defining variable types and more

### PR DESCRIPTION
**Pull request by simonschaufi**

### CODE REVIEW

File: Classes/Wizard/CoordinatepickerWizard.php [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <?php
-declare(strict_types = 1);
+
+declare(strict_types=1);
```
Comment (bobosch) [inline Classes/Wizard/CoordinatepickerWizard.php:3]: Removing white space? Better readable with

File: Classes/Updates/FileLocationUpdater.php [hunk 2]
```diff
@@ -106,8 +96,6 @@ public function getTitle(): string
 
     /**
      * Get description
-     *
-     * @return string Longer description of this updater
```
Comment (bobosch) [inline Classes/Updates/FileLocationUpdater.php:110]: Yes, not the best description, but it's not empty...

File: Classes/Controller/PluginController.php [hunk 3]
```diff
@@ -587,39 +585,40 @@ public function getMap()
                         )
                     ),
                 )
-                ->add('orderBy', 'FIELD(uid, ' . implode(',', $this->config['layer']) . ')', true)
+                #->add('orderBy', 'FIELD(uid, ' . implode(',', $this->config['layer']) . ')', true)
```
Comment (bobosch) [inline Classes/Controller/PluginController.php:588]: Does this break some functionality?

Comment (simonschaufi) [reply Classes/Controller/PluginController.php:588]: oh, well spotted. That only breaks in sqlite as sqlite does not support `FIELD`. I added this while creating tests. Will revert.

File: Classes/Wizard/CoordinatepickerWizard.php (hunk 1 above)
Comment (simonschaufi) [reply Classes/Wizard/CoordinatepickerWizard.php:3]: The core always uses the format `declare(strict_types=1);`

**Comment by simonschaufi** [comment]
@albig could you review please?

**Review by albig — APPROVED** [review]
Looks good to me. Done some functional tests on my development instance.

File: Classes/Backend/FormDataProvider/FlexFormManipulation.php [hunk 4]
```diff
@@ -40,9 +41,6 @@ class FlexFormManipulation implements FormDataProviderInterface
 {
     /**
      * Manipulate data of ods_osm_pi plugin.
-     *
-     * @param array $result
-     * @return array
```
[the remark this answers is not in the document]
Comment (bobosch) [reply Classes/Backend/FormDataProvider/FlexFormManipulation.php:45]: ... no new information there ... but yes, extending information would be even better ;-)

File: Classes/Wizard/VectordrawWizard.php [hunk 5]
```diff
@@ -23,9 +24,6 @@
  */
 class VectordrawWizard extends AbstractNode
 {
-    /**
-     * @return array
```
[the remark this answers is not in the document]
Comment (simonschaufi) [reply Classes/Wizard/VectordrawWizard.php:27]: done

File: Classes/Backend/FormDataProvider/FlexFormManipulation.php (hunk 4 above)
[the remark this answers is not in the document]
Comment (simonschaufi) [reply Classes/Backend/FormDataProvider/FlexFormManipulation.php:45]: done

**Review by albig — APPROVED** [review]

**Pull request merged**